src/features/pumpkinPlaza/components/BumpkinFriend.tsx

```diff
@@ -17,26 +21,38 @@ export const BumpkinFriend: React.FC<Props> = ({
   onClose,
 }) => {
   const [tab, setTab] = useState(0);
+  const { authService } = useContext(AuthProvider.Context);
+  const [authState, send] = useActor(authService);
+
+  const tabs = [
+    {
+      icon: SUNNYSIDE.icons.player,
+      name: "Player",
+    },
+    {
+      icon: SUNNYSIDE.icons.treasure,
+      name: "Shop",
+    },
+  ];
+
+  if (authState.context.token?.userAccess.admin) {
```

Comment:
only admins can kick players? Allow discord mods and beta pass users to at least mute them.

Also requiring a "mute/kick" reason is a great addition to prevent abuse.

Reply:
Yes, we will support moderators as well.

This will require a bit more work - hopefully we can get it done before the party in 2 days